From d4ead86b2184d1fc2748ed2b6fae8c0dceaf76c8 Mon Sep 17 00:00:00 2001
From: Ihar Hrachyshka <ihar.hrachyshka@gmail.com>
Date: Thu, 11 Sep 2025 23:37:28 -0400
Subject: [PATCH ovn 1/2] tests: Expect musl error string for EIO errno.

Musl uses a slightly different string representation for the error,
which makes the test fail on cleanup because an unexpected warning is
observed in the service log.

This patch will ignore both glibc and musl error messages.

Signed-off-by: Ihar Hrachyshka <ihar.hrachyshka@gmail.com>
---
 tests/ovn-controller.at | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tests/ovn-controller.at b/tests/ovn-controller.at
index 0b00906ae..bec06e527 100644
--- a/tests/ovn-controller.at
+++ b/tests/ovn-controller.at
@@ -846,6 +846,7 @@ OVN_CLEANUP([hv1
 /Certificate must be configured to use SSL/d
 /SSL_read: error/d
 /receive error: Input/d
+/receive error: I\/O error/d
 /connection dropped/d
 ])
 AT_CLEANUP
-- 
2.50.1

